### PR TITLE
Create a workflow that enables redfish alerting on  a dell node

### PR DESCRIPTION
**Pull request by tannoa2**

### CODE REVIEW

**Comment by tannoa2** [comment]
@RackHD/veyron @RackHD/corecommitters

File: lib/graphs/dell-configure-redfish-alerting-graph.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
+// Copyright 2017, EMC, Inc.
+
+"use strict";
+
+module.exports = {
+    "friendlyName": "Configure Dell Redfish Alerting",
+    "injectableName": "Graph.Configure.Redfish.Alerting",
```
Comment (anhou) [inline lib/graphs/dell-configure-redfish-alerting-graph.js]: all the tasks are Dell-related. so the injectableName include "Dell' is more accurate.

File: lib/graphs/dell-configure-redfish-alerting-graph.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+                },
+                properties: {},
+            },
+            ignoreFailure: true,
+            waitOn: {
+                'enable-redfish-alerts': 'finished'
+            }
+        },
+        {
+            "label": "redfish-subscribtion",
```
Comment (brianparry) [inline lib/graphs/dell-configure-redfish-alerting-graph.js]: @tannoa2 Should make this dependent

**Comment by tannoa2** [comment]
test this please

**Comment by tannoa2** [comment]
test this please

**Pull request merged**